app/src/pages/experiment/ExperimentCompareTable.tsx

```diff
@@ -93,6 +93,9 @@ export function ExperimentCompareTable(props: ExampleCompareTableProps) {
               error
               startTime
               endTime
+              trace {
+                id
+              }
```

Comment:
I'm not seeing where this is used.

Reply:
Oh yeah it's not, but I will. I needed roger's change.